src/sentry/runner/formatting.py

```diff
@@ -0,0 +1,106 @@
+# -*- coding: utf8 -*-
+
+from __future__ import absolute_import, print_function
+
+import re
+
+# Sentry colors taken from our design system. Might not look good on all
+# termianl themes tbh
```

Comment:
Is there a way to reuse terminal-defined colors as this seems pretty annoying for someone who fine-tuned their terminal colors for themselves. For instance people with color blindness (hi!) or people trying to avoid very bright colors etc?